rpi64/debimage-rpi64.yaml

```diff
@@ -0,0 +1,161 @@
+{{- $firmware_version := or .firmware_version "1.20200819" -}}
+{{- $mirror := or .mirror "http://deb.debian.org/debian" -}}
+{{- $suite := or .suite "bullseye" -}}
+{{- $kernel := or .kernel "linux-image-arm64" -}}
+{{- $cmdline := or .cmdline " rootwait ro fsck.mode=auto fsck.repair=yes console=ttyS1,115200" -}}
+{{- $image := or .image "debian-rpi64.img" -}}
+
+architecture: arm64
+
+actions:
+  {{ if ne $firmware_version "debian" }}
+  # Download all needed proprietary stuff first
+  - action: download
+    description: Download latest version of the firmware from GitHub
+    url: http://github.com/raspberrypi/firmware/archive/{{ $firmware_version }}.tar.gz
+    unpack: true     # Unpack downloaded file
+    name: firmware # name for usage directory with unpacked content in other actions
+  {{ end }}
+
+  - action: debootstrap
+    suite: {{ $suite }}
+    components:
+      - main
+      - non-free
+    mirror: {{ $mirror }}
+    variant: minbase
+
+  - action: apt
+    description: Install extra packages
+    packages:
+      - adduser
+      - bluetooth
+      - ca-cacert
+      - iproute2
+      - openssh-server
+      - sudo
+      - systemd-sysv
+      - u-boot-menu
+      - u-boot-rpi
+      - vim-tiny
+      - wget
```

Comment:
wget was likely added because of what you dropped in 1a490b3 so maybe wget should also be dropped?

Reply:
Thanks for the check!
I add it only because I want to have any ability to download anything from the internet out of the box. There is no curl or busybox in the minimal image.
Do you think `wget` is not needed?

Reply:
I don't have a strong opinion on this, just figured I would comment. I understand it might be convenient to have wget around, but it could also be argued that: 1/ some people prefer alternatives (curl, etc). 2/ if you have internet connectivity, you could just `apt-get install wget` and then use wget.... So I guess it basically boils down to what the purpose is for the debos-recipes repo. Is it providing minimal example recipes that people can extend for their own need? Is it more general recipes that are supposed to be useful as is (and should thus include more useful tools)? something else?... One example I'd personally like to see added is `dbus-user-session` (aka default-dbus-session). Dealing with systemd without having dbus is a PITA.

Reply:
Thanks! Have no strong opinion about `wget` as well. Added the `dbus-user-session` into the set of default packages.Anything else you think is needed? Otherwise I'll merge the current MR into master.

Reply:
All looks good to me. Could only spot these nitpicks.